### ISSUE TITLE
调试的时候遇到的，按F5就出现了

### ISSUE THREAD

**Issue by 5m1le71ger** [PowerShell/PSReadLine#3942]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
哎呀，出现问题。请在报告此 Bug 时添加以下详细信息。
在 GitHub 上报告: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
上 200 个密钥:
 h o n 3 1 1 \ p y t h o n . e x e ' Space ' c : \ U s e r s \ A S U S \ . v s c o d e \ e x t e n s i o n s \ m s - p y t h o n . d e b u g p y - 2 0 2 4 
. 0 . 0 - w i n 3 2 - x 6 4 \ b u n d l e d \ l i b s \ d e b u g p y \ a d a p t e r / . . / . . \ d e b u g p y \ l a u n c h e r ' Space ' 1 0 3 1 6 ' Space ' - - ' Space ' E : \ m y w o r k \ 砖 了 个 砖 \ b r i c k e d - a - b r i c k \ 格 子 截 图 工 具 . p y ' Space Enter


异常:
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。
参数名: top
实际值是 -1。
   在 System.Console.SetCursorPosition(Int32 left, Int32 top)
   在 Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   在 Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   在 Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   在 Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   在 Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

### Screenshot

![无标题](https://github.com/PowerShell/PSReadLine/assets/14155358/f99de963-7276-461f-8ba0-a4072b70a89b)

### Environment data

```console
PS Version: 5.1.19041.2364
PS HostName: ConsoleHost
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.3691 (WinBuild.160101.0800)
BufferWidth: 155
BufferHeight: 38
```

### Steps to reproduce

N/A

### Expected behavior

N/A

### Actual behavior

N/A

**Comment by StevenBucher98** [comment]
Thanks for the issue @5m1le71ger! We are going to need more detailed reproduction steps if we are to better help you resolve this issue, additionally it looks like you are on a pretty old version of PSReadLine so I suggest updating to the latest `Install-Module -Name PSReadLine` and seeing if the issue still happens.

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.3.4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.3.4) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
